node/task.ts

```diff
@@ -744,16 +744,30 @@ export function ls(options: string, paths: string[]): string[] {
  * @param     dest       destination path
  * @param     options    string -r, -f or -rf for recursive and force 
  * @param     continueOnError optional. whether to continue on error
+ * @param     retryCount optional. Retry count to copy the file
  */
-export function cp(source: string, dest: string, options?: string, continueOnError?: boolean): void {
+export function cp(source: string, dest: string, options?: string, continueOnError?: boolean, retryCount: number = 0): void {
```

Comment:
Could you please update docs as well? https://github.com/Microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#taskcp

Reply:
Thank you! Fixed it.